### DISCUSSION TITLE
Add loop/repeat button

### DISCUSSION THREAD

**Discussion by pmmontanari** [rekkyrosso/ampcast#7] [Ideas]
Hello, first of all, I have been having a blast with ampcast. The project is awesome. Since I discovered it some months ago I've been using on Windows and Linux without looking back.

One feature that I feel would be interesting is to loop songs. I wanted to loop a song and I could not find a loop or repeat button. I feel it would be a nice quality of life addition.

Anyway, again kudos and thanks for the superb work!

**Comment by rekkyrosso** [comment]
That does seem to be a standard thing to include. I'll try and add it for the next release.

**Comment by rekkyrosso** [comment]
This is now included in version 0.9.9.
